docs/src/further_topics/ugrid/operations.rst

```diff
@@ -0,0 +1,985 @@
+        ...     mesh_cubes = load(mesh_file)
+
+        # Extract a specific cube.
+        >>> mesh_cube1 = mesh_cubes.extract_cube("sea_surface_temperature")
+        >>> print(mesh_cube1)
+        sea_surface_temperature / (K)       (-- : 1; -- : 13824)
+            Mesh coordinates:
+                latitude                        -       x
+                longitude                       -       x
+            Auxiliary coordinates:
+                time                            x       -
+            Cell methods:
+                mean                        time (300 s)
+                mean                        time_counter
+            Attributes:
+                Conventions                 UGRID
+                description                 Created by xios
+                interval_operation          300 s
+                interval_write              1 d
+                name                        lfric_surface
+                online_operation            average
+                timeStamp                   2020-Feb-07 16:23:14 GMT
+                title                       Created by xios
+                uuid                        489bcef5-3d1c-4529-be42-4ab5f8c8497b
+
+        # Load the target grid.
+        >>> sample_grid = load_cube(grid_file)
+        >>> print(sample_grid)
+        sample_grid / (unknown)             (latitude: 180; longitude: 360)
+            Dimension coordinates:
+                latitude                             x               -
+                longitude                            -               x
+            Attributes:
+                Conventions                 CF-1.7
+
+        # Initialise the regridder.
+        >>> rg = MeshToGridESMFRegridder(mesh_cube1, sample_grid)
+
+        # Regrid the mesh cube cube.
+        >>> result1 = rg(mesh_cube1)
+        >>> print(result1)
+        sea_surface_temperature / (K)       (-- : 1; latitude: 180; longitude: 360)
+            Dimension coordinates:
+                latitude                        -            x               -
+                longitude                       -            -               x
+            Auxiliary coordinates:
+                time                            x            -               -
+            Cell methods:
+                mean                        time (300 s)
+                mean                        time_counter
+            Attributes:
+                Conventions                 UGRID
+                description                 Created by xios
+                interval_operation          300 s
+                interval_write              1 d
+                name                        lfric_surface
+                online_operation            average
+                timeStamp                   2020-Feb-07 16:23:14 GMT
+                title                       Created by xios
+                uuid                        489bcef5-3d1c-4529-be42-4ab5f8c8497b
+
```

Comment:
Again, it seems a slight shame not to _plot_ results, to show before+after,
but again it mixes up the example topics, which I guess you wanted to rule out ?

Reply:
@stephenworsley thoughts?

Reply:
TBH this might add some value but in the interests of time this feels like an 80-20 thing that we could get away without.

Reply:
yeah, pictures would be nice but no necessarily essential, and I think putting in the code to print them could distract from the regridding process.

Reply:
> manually keeping the images and code in sync,

:+1: Ok I didn't realise how this was being done.

I'm beginning to think this stuff should really be maintained somewhere else, with all the necessary dependencies pulled in, and perhaps using notebooks too.
A problem for another day !